app/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const {
+    getTopics,
+} = require('./controllers/topics')
+
+app.use(express.json());
```

Comment:
Please only add code that is required for use in the branch. This is not required by your app.server as of yet so shouldn't be added. (You have no endpoints that use the reqest object.)